doc/tutorial/argument_parser/index.md

```diff
@@ -59,7 +59,7 @@ The argument parser checks the following restrictions and throws a seqan3::parse
 * **Short identifiers**: must be unique and consist of only a single letter that is alphanumeric characters, `_` or `@`.
 * either the short or long id may be empty but not both at the same time.
 * Only the last positional option may be a list (see [lists](#section_list_positional_options)).
-* The flag identifiers `-h`, `--help`, `--advanced-help`, `--advanced-help`, `--export-help`, `--version`, `--copyright` are predefined and cannot be specified manually or used otherwise.
+* The flag identifiers `-h`, `--help`, `--advanced-help`, `--advanced-help`, `--export-help`, `--version`, `--copyright`, `--license` are predefined and cannot be specified manually or used otherwise.
```

Comment:
why do we have "advanced-help" twice? :thinking:

Reply:
:eyes: because it is super important and I never make errors.. (as well as my reviewers)

@tloka Can you change the firsst `--advanced-help` to `-hh` ?

Reply:
I will in case you finally decide to merge this PR.
Otherwise I can do it in a seperate PR.